app/controllers/projects_controller.rb

```diff
@@ -318,8 +322,8 @@ def fork
       redirect_to child.urlbase
       # todo - notifications
     else
-      flash[:alert] = "Couldn't fork project, try again."
-      redirect_to @project
+      flash[:alert] = "Couldn't fork project, you already have a project with the same name."
```

Comment:
@Alwahsh it could happen because of a server failure - i.e. couldn't connect to the database; undiagnosed server bug; etc